src/assets/sass/components/ae-container/structure.scss

```diff
@@ -37,6 +37,7 @@
     }
 
     label {
+        color: white;
```

Comment:
I think we usually write this as `#fff` elsewhere?

Reply:
Yes ... good catch

![](https://media.giphy.com/media/bsEUpxQP9Spj2/giphy.gif)